0000-py-version-support.md

```diff
@@ -0,0 +1,244 @@
+|------------|---------------------|----------------|
+|   10.13    |    September 2020   |    2.7         |
+|   10.14    |    September 2021   |    2.7         |
+|   10.15    |    TBD              |    2.7         |
+|   10.16    |    TBD              |   no default   |
+
+We have always included the Python binary with the mac packages, so that will not change for
+users that use the mac Python packages. Starting with the Sodium release we will no longer
+support Python 2.7, so if users install Salt on OSX using pip or any other means they will
+need to install Python >= 3.5.
+
+Fedora
+------
+Fedora releases a new version every 6 months and supports these releases for approximately 13 months
+. Our current support mirrors that of the upstream and they are currently supporting Fedora 30, 31 and
+32. They do not have current EOL dates for these releases yet here:
+https://endoflife.software/operating-systems/linux/fedora
+
+|     OS     |    End of Life              |  Default Python |
+|------------|-----------------------------|-----------------|
+|     30     |    TBD (possibly june 2020) |      3.7        |
+|     31     |    TBD (possibly oct 2020)  |      3.7        |
+|     32     |    TBD (possibly apr 2020)  |      3.8        |
+
+Based off of these dates and Python's End of Life dates, Fedora will not
+be of any concern with this change in policy. Fedora releases fairly regularly
+and includes the latest Python version. Both packages and pip installs
+will work against supported Fedora versions going forward.
+
+SLES/OpenSuse
+----
+The General End of Life dates do conflict with the current Python Version
+End of Life dates. Since Suse does the packaging here they have been informed
+of this policy change and will include a resolution around this with their packages
+when these version conflicts occur.
+
+Other Linux
+-----------
+
+|     OS     |    End of Life      | Default Python |
+|------------|---------------------|----------------|
+|     Rhel 6 |    30 Nov 2020      |      2.6       |
+|     Rhel 7 |    30 Jun 2024      |      2.7       |
+|     Rhel 8 |    05 May 2029      |      None      |
+|     Ubuntu 16.04  |    April 2021    |    2.7/3.5 |
+|     Ubuntu 18.04  |    April 2023    |    3.6     |
+|     Ubuntu 20.04  |    2030          |    3.8     |
+|     Debian 8      |    30 June 2020  |  2.7, 3.4  |
+|     Debian 9      |    June 2022     |  2.7, 3.5  |
+|     Debian 10      |    June 2024    |  2.7, 3.7  |
+
+Based on this chart above all of these Operating Systems will or already have conflicted with
+the Python's End of Life Support dates. We will need to include Python in our packages for these
+Operating Systems especially at any given time when their default Python version is considered
+End of Life. This does impact users who would want to pip install Salt. They will need to install
+a newer version of Python before they can install Salt.
+
+Other Reasonable-Effor Support Linux
```

Comment:
Effort

Reply:
s/Linux/Operating Systems/

(FreeBSD and SmartOS aren't Linuxes)